prow/plugins/approve/approvers2/owners.go

```diff
@@ -404,45 +477,34 @@ func (ap Approvers) GetNoIssueApproversSet() sets.String {
 // GetFilesApprovers returns a map from files -> list of current approvers.
 func (ap Approvers) GetFilesApprovers() map[string]sets.String {
 	filesApprovers := map[string]sets.String{}
-	currentApprovers := ap.GetCurrentApproversSetCased()
-	for ownersFilename, potentialApprovers := range ap.owners.GetApprovers() {
-		// The order of parameter matters here:
-		// - currentApprovers is the list of github handles that have approved
-		// - potentialApprovers is the list of handles in the OWNER
-		// files (lower case).
-		//
-		// We want to keep the syntax of the github handle
-		// rather than the potential mis-cased username found in
-		// the OWNERS file, that's why it's the first parameter.
-		filesApprovers[ownersFilename] = CaseInsensitiveIntersection(currentApprovers, potentialApprovers)
+	for fn, potentialApprovers := range ap.owners.GetApprovers() {
+		// potentialApprovers is the list of github handles who can approve file fn.
+		filesApprovers[fn] = approversForFile(fn, potentialApprovers, ap.approvers)
```

Comment:
The `approversForFile` already takes care of case sensitivity. We don't directly take the intersection because we also need to check if the approval actually covers the file.

Reply:
OK I think you're right, this was just a very specific comment you deleted so I wanted to make sure you're still handling this corner case (mis-cased OWNERS file vs. github login)